whereby/sdk
# common: Drop sending of `selfId` in `join_room` signal request

### Description

The `selfId` parameter is no longer required in the `join_room` request so remove it from the request payload also.

### Testing

1. Run `yarn dev`
2. In the developer console, simulate a network disconnect/reconnect
3. Ensure that the client reconnects to the call on reconnect

### Checklist

        under the terms of the MIT license.

### Additional Information
